### PR TITLE
choose leader for a team

### PR DESCRIPTION
**Pull request by PhuongTrieu**

### CODE REVIEW

File: app/controllers/admin/teams_controller.rb [hunk 1]
```diff
@@ -15,6 +16,7 @@ def new
   end
 
   def create
+    byebug
```
Comment (vjnhat) [inline app/controllers/admin/teams_controller.rb]: ????

File: app/views/admin/teams/show.html.erb [hunk 2]
```diff
@@ -1,3 +1,19 @@
 <div class="row">
+  <div class="col-md-4" id="team-detail">
+    <h3><%= @team.name %></h3>
+    <p><%= t("team.created_at") %> <%= @team.created_at %></p>
+    <p><%= t("team.description") %> <%= @team.description %></p>
+    <p><%= t("team.leader_member") %>
```
Comment (vjnhat) [inline app/views/admin/teams/show.html.erb]: remove ()

File: app/views/admin/teams/show.html.erb [hunk 3]
```diff
@@ -1,3 +1,19 @@
 <div class="row">
+  <div class="col-md-4" id="team-detail">
+    <h3><%= @team.name %></h3>
+    <p><%= t("team.created_at") %> <%= @team.created_at %></p>
```
Comment (vjnhat) [inline app/views/admin/teams/show.html.erb]: remove ()

File: app/views/shared/_add_users.html.erb [hunk 4]
```diff
@@ -1,16 +1,3 @@
...
-  <h4><%= t("team.users") %></h4>
-  <ul class="jumbotron">
-    <div id="team_users">
-      <%= render "shared/users_in_team" %>
-    </div>
-  </ul>
-  <%= link_to t(:back), :back %>
-</div>
 <div class="col-md-8" id="add-users-content">
 <h3><%= t("team.add_users") %></h3>
```
Comment (vjnhat) [inline app/views/shared/_add_users.html.erb]: remove ()

File: app/views/admin/teams/show.html.erb [hunk 5]
```diff
@@ -1,3 +1,19 @@
 <div class="row">
+  <div class="col-md-4" id="team-detail">
+    <h3><%= @team.name %></h3>
+    <p><%= t "team.created_at" %> <%= @team.created_at %></p>
+    <p><%= t "team.description" %> <%= @team.description %></p>
+    <p><%= t "team.leader_member" %>
+      <%= link_to @leader.name, admin_user_path(@leader) %>
```
Comment (vjnhat) [inline app/views/admin/teams/show.html.erb]: try remove () in this line if it run normally

File: app/views/admin/teams/show.html.erb [hunk 6]
```diff
@@ -1,3 +1,19 @@
...
+    </p>
+    <p><%= link_to t(:edit), edit_admin_team_path(@team) %></p>
```
Comment (vjnhat) [inline app/views/admin/teams/show.html.erb]: same above

File: app/views/admin/teams/show.html.erb (hunk 5 above)
Comment (PhuongTrieu) [inline app/views/admin/teams/show.html.erb]: It can not run if remove ()

File: app/views/admin/teams/show.html.erb (hunk 6 above)
Comment (PhuongTrieu) [inline app/views/admin/teams/show.html.erb]: Can't run!

File: app/controllers/admin/teams_controller.rb [hunk 7]
```diff
@@ -8,6 +8,7 @@ def index
 
   def show
     @team = Team.find params[:id]
+    @leader = User.find @team.leader_id
```
Comment (dieunb) [inline app/controllers/admin/teams_controller.rb]: why don't `@leader = @team.leader`

File: app/models/user.rb [hunk 8]
```diff
@@ -13,4 +13,5 @@ class User < ActiveRecord::Base
   accepts_nested_attributes_for :skill_users, allow_destroy: true
 
   scope :users_not_in_team, ->{where(Settings.sql.user.team_users_scope)}
+  scope :normal_users, ->{where(Settings.sql.user.normal_users)}
```
Comment (dieunb) [inline app/models/user.rb]: remove `()`

File: app/controllers/admin/teams_controller.rb [hunk 9]
```diff
@@ -26,6 +27,7 @@ def create
 
   def edit
     @team = Team.find params[:id]
+    @leader = @team.users.pluck(:name, :id)
```
Comment (dieunb) [inline app/controllers/admin/teams_controller.rb]: remove `()`

File: app/models/team.rb [hunk 10]
```diff
@@ -2,7 +2,13 @@ class Team < ActiveRecord::Base
   has_many :users, through: :team_users, dependent: :destroy
   has_many :team_users
 
+  belongs_to :leader, class_name: "User", foreign_key: :leader_id
+
   accepts_nested_attributes_for :users, allow_destroy: true
 
   validates :name, presence: true
+
+  def create_leader id
```
Comment (dieunb) [inline app/models/team.rb]: what do this function do?

**Pull request merged**